.travis.yml

```diff
@@ -3,20 +3,18 @@ dist: bionic
 cache: bundler
 
 rvm:
-  - 2.3
-  - 2.4
   - 2.5
   - 2.6
   - 2.7
+  - 3.0
 
 before_install:
   - gem update --system -N > /dev/null && echo "Rubygems version $(gem --version)"
-  - if [ $TRAVIS_RUBY_VERSION == 2.7 ]; then gem install rubocop:1.3.1; fi
 
 script:
   - bundle exec rspec --format progress
   - bundle exec cucumber --format progress
-  - if [ $TRAVIS_RUBY_VERSION == 2.7 ]; then rubocop; fi
+  - bundle exec rubocop
```

Comment:
I guess there was a reason not to run rubocop on every job ?

Reply:
mostly because it was broken with ruby 2.3
no major issue running with all version has long as it passes :heavy_check_mark: